LICENSE

```diff
@@ -1,7 +1,6 @@
 License (MIT)
--------------
 
-Copyright (c) 2016 Jeff Escalante, Carrot Creative
+Copyright (c) PostHTML
```

Comment:
You can't remove my name from the license, I wrote this code. You can add posthtml to the names though!